### DISCUSSION TITLE
List group

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#724] [Proposals]
Últimamente existe la necesidad de poder agrupar listados

Actualmente no existe una buena solución para hacer esto en Mística y lo ideal sería que el componente rowList admitiera agrupar visualmente con un boxed.

Una posible solución sencilla sería evolucionar el componente `<RowList>` y añadirle una prop `<RowList Boxed>`

Un problema que yo veo aquí es que no es lo mismo esto
![image](https://user-images.githubusercontent.com/6722153/186647873-5f1c763e-26a9-487d-8eb6-fb20ec4294e8.png)

que esto
![image](https://user-images.githubusercontent.com/6722153/186647933-a74e6640-d5d2-4756-a7aa-4e823b7cfa12.png)

el primer caso es claramente una agrupación de listas mientras que el segundo caso se acerca a una card (o más bien se aleja de una lista como tal)

**Comment by MontseTelefonica** [comment]
Esto sería bastante importante para el proyecto de Start tab UK. Sin meter presión :P Además considero que es algo muy positivo que se puede aplicar a otros casos de uso de otras OBs.

**Comment by yceballost** [comment]
Dev hay hecho una prueba y parece que esto tiene sentido habilitarlo en Mística.
https://github.com/Telefonica/mistica-web/pull/535

https://ibit.ly/QFp9

**Comment by yceballost** [comment]
Incluiremos esto en breves en Mística @MontseTelefonica podríamos decir que en diseño ya podría plantearse

**Comment by MontseTelefonica** [comment]
Súper! Gracias!
